### PR TITLE
Get purchase {controller, routes, i18/en/index.js}

### PR DESCRIPTION
**Pull request by jonwk-19**
Get purchase

### CODE REVIEW

File: src/application/purchase/purchase.controller.js [hunk 1]
```diff
@@ -12,6 +12,125 @@ import { ProductNotEnoughStock } from "./purchase.errors.js";
 import { getMoneyExchangeRate } from "./purchase.util.js";
 import { AccountInsufficientFundsError } from "../account/account.error.js";
 
+export const getAllPurcahse = async (req, res) => {
```
Comment (bcastillo-2022474) [inline src/application/purchase/purchase.controller.js:15]: fix typo

**Review by bcastillo-2022474 — APPROVED** [review]
LGTM

File: src/application/purchase/purchase.controller.js (hunk 1 above)
Comment (bcastillo-2022474) [reply src/application/purchase/purchase.controller.js:15]:
```suggestion
export const getAllPurchase = async (req, res) => {
```

**Comment by jonwk-19** [comment]
adfsadf

**Pull request merged**